src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/ServiceHearingsControllerTest.java

```diff
@@ -66,13 +97,39 @@ void testPostRequestServiceHearingValues() throws Exception {
 
         given(serviceHearingsService.getServiceHearingValues(request))
             .willReturn(ServiceHearingValues.builder()
+                .caseDeepLink(null)
+                .caseManagementLocationCode("")
+                .caseRestrictedFlag(true)
+                .caseSlaStartDate("")
+                .externalCaseReference("")
+                .hearingChannels(null)
```

Comment:
do you have to include this if you're just setting it to null? you can just omit it. probably same with empty string, no?